tests/cases/fourslash/codeFixPromiseToAsync.ts

```diff
@@ -0,0 +1,22 @@
+/// <reference path='fourslash.ts' />
+
+// @target: es6
+
+////function [|f|](): Promise<void> {

```

Comment:
You might want to have a look at the way we test Extract Function (grab me for details).  This will be a little different, since it's a code fix rather than a refactoring, but there's something to be said for re-generatable baselines.

Reply:
Yes, this should use baseline machinery I think.